### PR TITLE
today's PR

### PR DESCRIPTION
**Pull request by Marthatwesi**
I added all codes lost

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+app.use(express.json());
+app.use(morgan("dev"));
+
+
+const port = process.env.PORT || 4000;
+
+
+try {
+ 
+  mongoose.connect("mongodb://localhost:27017/marthadb", { useNewUrlParser: true });
```
Comment (Eli250) [inline src/app.js]: @Marthatwesi do not expose your database URL to the public. Use environment variables.
Also, remove unnecessary empty lines.

File: src/controller/articleController.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        }
+    } 
+    catch (error) {
+            console.log(error);
+            return res.status(500).json({ error: "The internal Server is facing some errors" });
+          }
+
+   
+    }
+
```
Comment (Eli250) [inline src/controller/articleController.js:42]: Remove empty lines.

File: src/controller/editArticleController.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                    public_id: imgResult.public_id,url: imgResult.secure_url
+                }
+            }
+            await Article.findOneAndUpdate({_id: req.params.id})
+
+            res.status(200).json({data})
+
+        }
+        
+
```
Comment (Eli250) [inline src/controller/editArticleController.js:32]: Remove empty lines.

File: src/controller/getOneArticle.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+// import Article from "../model/article";
```
Comment (Eli250) [inline src/controller/getOneArticle.js:1]: Delete this file if you are not using it.

File: src/routes/api/authRoutes.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+)
+
+
+export default router;
+
+
+
+
+
+// import express from "express";
```
Comment (Eli250) [inline src/routes/api/authRoutes.js]: Remove all these comments along with all empty lines.

**Review by Eli250 — CHANGES_REQUESTED** [review]
Work all comments.

**Comment by Marthatwesi** [comment]
Okay, I will do that. Thanks

**Pull request merged**